charts/datarepo/Chart.yaml

```
@@ -15,6 +15,8 @@ type: application
 
 version: 0.1.815
 appVersion: 0.1.815
+version: 0.2.0
+appVersion: 0.2.0
```

Is this correct? I'd expect the new version / appVersion to replace the old one
```suggestion
version: 0.2.0
appVersion: 0.2.0
```

---

Ah that's my mistake from resolving merge conficts 🤦‍♀️